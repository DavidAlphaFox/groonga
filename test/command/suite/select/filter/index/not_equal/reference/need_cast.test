table_create Types TABLE_PAT_KEY ShortText

table_create Entries TABLE_NO_KEY
column_create Entries type COLUMN_SCALAR Types

column_create Types entries COLUMN_INDEX Entries type

load --table Entries
[
{"type": "10"},
{"type": "29"},
{"type": "fast"}
]

select Entries --filter 'true && type != 29'
