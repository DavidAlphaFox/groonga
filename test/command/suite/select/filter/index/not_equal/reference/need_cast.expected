table_create Types TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
table_create Entries TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Entries type COLUMN_SCALAR Types
[[0,0.0,0.0],true]
column_create Types entries COLUMN_INDEX Entries type
[[0,0.0,0.0],true]
load --table Entries
[
{"type": "10"},
{"type": "29"},
{"type": "fast"}
]
[[0,0.0,0.0],3]
select Entries --filter 'true && type != 29'
[[0,0.0,0.0],[[[2],[["_id","UInt32"],["type","Types"]],[1,"10"],[3,"fast"]]]]
